packages/analytics/analytics-metric-provider/src/components/display/MetricsCard.vue

```diff
@@ -260,43 +257,43 @@ $row-gap-size: 12px;
 
   &-title {
     align-items: center;
-    color: var(--kong-ui-metric-card-title, $kui-color-text);
+    color: var(--kui-color-text, $kui-color-text);
     display: flex;
     flex-direction: row;
-    font-size: $kui-font-size-30;
-    font-weight: $kui-font-weight-medium;
-    line-height: $kui-line-height-20;
-    margin: $kui-space-0;
+    font-size: var(--kui-font-size-30, $kui-font-size-30);
+    font-weight: var(--kui-font-weight-medium, $kui-font-weight-medium);
+    line-height: var(--kui-line-height-20, $kui-line-height-20);
+    margin: var(--kui-space-0, $kui-space-0);
 
     // The metric card title is always 14px; the "small" variant is the exception
     &.sm {
       font-size: $kui-font-size-20;
```

Comment:
Anywhere you're using a SCSS variable, you should have the `var()` syntax as well

Reply:
Will update. Var syntax even for `kui-breakpoint-***`? 

It can be overridden, but maybe in this case it shouldn't be (due to unforeseen bad visual things happening)?

Reply:
You can’t use the custom properties in media query declarations, so just stick to SCSS bars there